src/ui/common/src/components/operators/WithOperatorHeader.tsx

```diff
@@ -0,0 +1,143 @@
+import {
```

Comment:
This wrapper renders core page components for an operator. Mostly for page title and `ArtifactSummaryList`. This is shared in op, check, and metrics detail page.